daliuge-translator/dlg/dropmake/lg_node.py

```diff
@@ -100,52 +100,6 @@ def __init__(self, jd, group_q, done_dict, ssid):
     # def __str__(self):
     #     return json.dumps(self.jd)
 
-    @property
```

Comment:
These are being removed because there were duplicate definitions below. As Python is interpreted, these definitions would have been over-written and so the latter definitions are the ones we have been using anyway.